include/flatbuffers/flexbuffers.h

```diff
@@ -743,7 +745,7 @@ inline Reference GetRoot(const uint8_t *buffer, size_t size) {
 }
 
 inline Reference GetRoot(const std::vector<uint8_t> &buffer) {
-  return GetRoot(buffer.data(), buffer.size());
+  return GetRoot(flatbuffers::vector_data(&buffer), buffer.size());
```

Comment:
We're replacing the STL, so STL style > Google style in this case. So both `&` would be best. Failing that, just the const one.